bitcoin/types.go

```diff
@@ -85,35 +85,53 @@ const (
 	P2PKHScriptPubkeySize = 25               // P2PKH size
 )
 
+// CreateMainNetParams is a function to override default mainnet settings with address prefixes
+func CreateMainNetParams() (*chaincfg.Params) {
```

Comment:
@sidhujag would it make more sense to provide a JSON file at initialization that determines the values of `chaincfg.Params`? If not provided, it would just default to what we already had here.

In the general case, I think this would mean that a fork could integrate without maintaining their own repo?

Reply:
Yup that makes sense. Outside of these changes and the default table/log sizes for badger we should be able to just use rosetta-bitcoin. I actually prefer this option. If we can move it out to json then we can configure network settings, the genesis details, the address prefixes and it should just work.